optuna/pruners/hyperband.py

```diff
@@ -59,6 +62,10 @@ class HyperbandPruner(BasePruner):
             :math:`\\eta` in the paper. See the details for
             :class:`~optuna.pruners.SuccessiveHalvingPruner`.
         n_brackets:
+
+            .. deprecated:: 1.4.0
+                This input was removed from class:`~optuna.pruners.HyperbandPruner`.
```

Comment:
I feel that `input` is a bit ambiguous. How about using `argument`?
And also, `n_brackets` is deprecated but has not been removed yet.
```suggestion
                This argument will be removed from class:`~optuna.pruners.HyperbandPruner`.
```

Reply:
Since `HyperbandPruner` has been experimental, I would say that it's acceptable to remove this argument when the next major release comes out.